bedRModv2.tex

```diff
@@ -0,0 +1,379 @@
+        \toprule
+        Col & \acs{bedRMod} Field & Type & Regex or range & Brief description \\
+        \midrule
+        1
+        & \textsf{chrom}
+        & String
+        & \texttt{[[:alnum:]\_]\{1,255\}}\footnote{\texttt{[[:alnum:]\_]} is equivalent to the \ac{regex}~\texttt{[A-Za-z0-9\_]}. % chktex 8
+        It is also equivalent to the Perl extension~\texttt{[[:word:]]}}
+        & Chromosome name \\
+        2 & \textsf{chromStart} & Int & $[0, 2^{64}-1]$ & \textbf{Feature} start position \\
+        3 & \textsf{chromEnd} & Int & $[0, 2^{64} -1]$ & \textbf{Feature} end position \\
+        4 
+        & \textsf{name} 
+        & String 
+        & \texttt{[{\textbackslash}x20-{\textbackslash}x7e]\{1,255\}} 
+        & Modification name \\
+        5 & \textsf{score} & String & \texttt{[{\textbackslash}x20-{\textbackslash}x7e]\{1,255\}} & Modification confidence \\
+        6 & \textsf{strand} & String & \texttt{[-+.]} & \textbf{Feature} strand \\
+        7 & \textsf{thickStart} & Int & $[0, 2^{64}-1]$ & Thick start position, typically same as~\textsf{chromStart} \\
+        8 & \textsf{thickEnd} & Int & $[0, 2^{64}-1]$ & Thick end position, typically same as~\textsf{chromEnd} \\
+        9 & \textsf{itemRgb} & Int,Int,Int & \texttt{(}$[0, 255], [0,255], [0,255]$\texttt{) | 0} & Display color \\ % chktex 9
+        10 & \textsf{coverage} & Int &  $[0, 2^{64}-1]$ & Coverage \\
+        11 & \textsf{frequency} & Float & $(0, 100]$ & Percentage of modification \\
+        \bottomrule
+      \end{tabularx}
+    \end{adjustwidth}
+    \caption{\textbf{\acs{bedRMod} Fields.}}\label{tab:fields}
+  \end{table}
+\end{savenotes}
+
+In a \ac{bedRMod}~\textbf{file}, each~\textbf{data line} must have the same number of~\textbf{field}s.
+The positions in~\textbf{\acs{bedRMod} field}s are all described in the~\textbf{0-based, half-open coordinate system}, exactly as 
+described in the official \ac{BED} specifications.
+
+\subsection{Coordinates}
+Refer to the official \ac{BED} specifications.
+
+\subsection{Simple attributes}
+\begin{enumerate}
+\item \textsf{name}: String that describes the~\textbf{feature}, \textit{i.e.} the modification. \textsf{name} must describe 
+the modification using the \emph{short name} of the MODOMICS nomenclature\footnote{MODOMICS, \url{https://www.genesilico.pl/modomics/modifications}}, or the base modification code described in the SAMtags, or a numeric ChEBI code. The MODOMICS \emph{short name} corresponding to~\textsf{name} is always described in the~\textbf{header field}~\textsf{modification\textunderscore names}.
+
+\item \textsf{score}: String representation of the confidence in calling this modification. A value of 0 indicates missing data or uninformative~\textsf{score}. Any measure of confidence is valid, but a \ac{bedRMod}~\textbf{file} with non-integer-like~\textsf{score} values outside the range $[0, 1000]$ may fail from being correctly displayed in a visual representation.
```

Comment:
Technically this is compatible with the bedMethyl schema where score is `valid_coverage`. However the maximum value can be higher than 1000 (up to the max depth allowed by htslib which is `2147483647` last I checked). Is there a particular reason for 1000? Maybe the meaning of the score should be annotated in a header field. For example, `modkit bedmethyl merge` should fail to merge tables when the scores have different meanings (or handle it somehow).

Reply:
`score` is `String [\x20-\x7e]{1,255}`, so could in theory include any character. This allows to represent `valid_coverage`, as well as eventually any other measure of confidence. In Section 3.3, we recommend that `valid_coverage` should be used (and this is what we will specify in Sci-ModoM). But for euf-specs, the definition should remain general enough. If `score` has a special meaning, one can always use `#bioinformatics_workflow` to describe it.

The last statement merely describes potential interoperability issues with the BED format, where `score` is `Int [0, 1000]`. This is only a "disclaimer", *cf* [bedtools score](https://bedtools.readthedocs.io/en/latest/content/general-usage.html?highlight=bed%20format).

Reply:
Sounds good.